app/src/main/res/values/strings.xml

```diff
@@ -1685,4 +1685,8 @@
     <string name="onboarding_firefox_account_auto_signin_header_2">You are signed in as %s on another Firefox browser on this phone. Would you like to sign in with this account?</string>
     <!-- Deprecated: Describes the add to homescreen functionality -->
     <string name="add_to_homescreen_description">You can easily add this website to your phone’s Home screen to have instant access and browse faster with an app-like experience.</string>
+    <!--Content Description for webAddressView -->
```

Comment:
nit: Missing space after `<!--` and we typically describe the UI element when in the comments instead of using their IDs.

```suggestion
    <!-- Content Description for a login's web address text field -->
```

Reply:
@gabrielluong I have made the changes